hapi-proto/src/main/java/com/hederahashgraph/builder/TransactionSigner.java

```diff
@@ -307,8 +191,8 @@ public static SignatureMap signAsSignatureMap(byte[] messageBytes, List<Key> key
    * @return found minimum prefix length
    */
   private static int findMinPrefixLength(Set<Key> keys) {
-    if(keys.size() == 1) {
-      return 0;
+    if (keys.size() == 1) {
+      return 1;
```

Comment:
I remember the HAPI API doc says when 0 is used as the prefix length, it means only one public key is used to sign. Will this change have any impact later?

Reply:
```
The client may use any number of bytes from 0 to the whole length of the public key for pubKeyPrefix.
If 0 bytes is used, then it is assumed that only one public key is used to sign.
```
`TransactionSigner` is a class that helps clients sign their transactions. So it can use 1 byte of the public key for `pubKeyPrefix` as suggested. Using 0-byte `pubKeyPrefix` means the `pubKeyPrefix` field is empty which causes some issue in parsing, as I remember why I made this change. Also the doc here should be updated to read `... that only one private key is used to sign.`

Reply:
The point is not on the TransactionSigner class, it's on the generated SignaturePair which will be used as part of the signed transaction. As long as both client and server sides understand this, I think it would be fine.